zstackctl/zstackctl/ctl.py

```
@@ -11439,6 +11439,16 @@ def status(self):
         )
         info_and_debug(format_str)
 
+    def ensure_iptables_accept_port(self, port):
+        cmd = (
+                  'iptables-save | grep -- "-A INPUT -p tcp -m tcp --dport %s -j ACCEPT" > /dev/null '
+                  '|| (iptables -I INPUT -p tcp -m tcp --dport %s -j ACCEPT && service iptables save)'
+              ) % (port, port)
+        shell_no_pipe(cmd)
+        if self.zsha2_utils:
+            self.zsha2_utils.execute_on_peer(cmd);
```

Ruff (0.12.2)

Statement ends with an unnecessary semicolon

(E703)